jgrapht-core/src/test/java/org/jgrapht/AllTests.java

```diff
@@ -32,8 +30,8 @@
  * @author Barak Naveh
  */
 @RunWith(Suite.class)
-@Suite.SuiteClasses({ AllAlgTests.class, AllAlgUtilTests.class, AllGenerateTests.class,
-    AllGraphTests.class, AllTraverseTests.class, AllUtilTests.class })
+@Suite.SuiteClasses({ AllAlgTests.class, AllGraphTests.class, AllTraverseTests.class,
```

Comment:
New versions of Eclipse do not really need this. You can run tests by right-clicking on the package and using `Run As->JUnit Test`. Maven test also does not care. 

If you actually run all the tests from Eclipse, it finds the tests twice (a) from the suites and (b) from the packages. It is clever enough, though, to not execute them twice.

I have not gone into removing them all, but I am not keeping them updated.